src/backend/distributed/planner/multi_router_planner.c

```diff
@@ -535,10 +535,21 @@ MasterIrreducibleExpressionWalker(Node *expression, WalkerState *state)
 	 * Once you've added them to this check, make sure you also evaluate them in the
 	 * executor!
 	 */
-	StaticAssertStmt(PG_VERSION_NUM < 90600, "When porting to a newer PG this section"
```

Comment:
Yeah, I realized that and read over it to make sure I didn't miss any types, but until 9.5 is gone, might as well keep this block the way we wrote it.